src/main/scala/nl/knaw/dans/easy/license/CommandLineOptions.scala

```diff
@@ -15,35 +15,96 @@
  */
 package nl.knaw.dans.easy.license
 
-import java.io.{File, PrintWriter}
-import java.net.URL
+import java.io.File
+import javax.naming.Context
+import javax.naming.ldap.InitialLdapContext
 
-import org.rogach.scallop.ScallopConf
+import com.yourmediashelf.fedora.client.FedoraCredentials
+import org.apache.commons.configuration.PropertiesConfiguration
+import org.rogach.scallop._
+import org.slf4j.LoggerFactory
 
-class CommandLineOptions (args: Array[String]) extends ScallopConf(args) {
-  printedName = "easy-license-creator";
-  val __________ = " " * printedName.size
+class CommandLineOptions(args: Array[String]) extends ScallopConf(args) {
+  
+  appendDefaultToDescription = true
+  editBuilder(_.setHelpWidth(110))
+
+  val fileMayNotExist = singleArgConverter(new File(_))
+
+  printedName = "easy-license-creator"
 
   version(s"$printedName v${Version()}")
   banner(s"""
-           |<Replace with one sentence describing the main task of this module>
+           |Create a license for the given datasetID and depositorID. The latter is optional as it can also
+           | be retrieved from the datasetID. The license will be saved at the indicated location.
            |
            |Usage:
            |
-           |$printedName <synopsis of command line parameters>
-           |${__________} <...possibly continued here>
+           |$printedName <datasetID> <license-file>
            |
            |Options:
            |""".stripMargin)
-  //val url = opt[String]("someOption", noshort = true, descr = "Description of the option", default = Some("Default value"))
-  footer("")
+
+  val datasetID = trailArg[DatasetID](name = "dataset-id",
+    descr = "The ID of the dataset of which a license has to be created")
+
+  val outputFile = trailArg[File](name = "license-file",
+    descr = "The file location where the license needs to be stored")(fileMayNotExist)
+
+  verify()
 }
 
 object CommandLineOptions {
 
+  val log = LoggerFactory.getLogger(getClass)
+
   def parse(args: Array[String]): Parameters = {
+    log.debug("Loading application properties ...")
+    val homeDir = new File(System.getProperty("app.home"))
+    val props = {
+      val ps = new PropertiesConfiguration()
+      ps.setDelimiterParsingDisabled(true)
+      ps.load(new File(homeDir, "cfg/application.properties"))
+
+      ps
+    }
+
+    log.debug("Parsing command line ...")
     val opts = new CommandLineOptions(args)
-    // Fill Parameters with values from command line
-    Parameters()
+
+    val params = Parameters(
+      appHomeDir = homeDir,
+      templateDir = new File(homeDir, "res/"),
+      outputFile = opts.outputFile(),
+      datasetID = opts.datasetID(),
+      vagrant = {
```

Comment:
Not so happy about this part. Now the application code contains development environment specific stuff and I think that is unnecessary. What I think we should implement is:

1. A configuration setting for the command that creates the pdf, e.g. `htm2pdf.cmd`
2. Implement a bash script that implements this command by doing the ssh stuff that is now inside the scala code, roughly (for development you may assume that the developer changes the script if his environment is not completely standard, for instance if `easy-dtap` is in a different location):
                     
        #!/bin/bash
        ssh -oStrictHostKeyChecking=no -i ~/git/dtap/easy-dtap/.vagrant... deasy.dans.knaw.nl  \
              weasyprint -e utf8 -f pdf - -
3. Create a second script that calls weasyprint on localhost.